services/spar/src/Spar/Sem/IdPConfigStore/Mem.hs

```diff
@@ -46,7 +46,9 @@ idPToMem = evState . evEff
     evEff = reinterpret @_ @(State TypedState) $ \case
       InsertConfig iw ->
         modify' (insertConfig iw)
-      NewHandle _tid -> pure $ IdPHandle "IdP 1" --todo(leif): generate a new handle
+      NewHandle _tid ->
+        -- Same handle for all IdPs is good enough, for now
```

Comment:
Can we elaborate on this? Why is a hardcoded handle fine?

Reply:
Because this is just a label shown to the team admin in the UI. IMO uniqueness is irrelevant for tests,

Reply:
:+1: we'll see right away if this is not enough any more in the future, and then we can build something more sophisticated.